Packs/MandiantAdvantageThreatIntelligence/Integrations/FeedMandiant/FeedMandiant.yml

```
@@ -0,0 +1,134 @@
+  type: 0
+  required: false
+- additionalinfo: Indicators from this integration instance will be marked with this reputation.
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  type: 18
+  required: false
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- name: feedMinimumThreatScore
+  type: 0
+  display: Feed Minimum Threat Score
+  required: true
+  defaultvalue: 80
+  additionalinfo: 'The minimum Threat Score value to import as part of the feed'
+- additionalinfo: The maximum value allowed is 90 days.
+  defaultvalue: 30
+  display: First fetch time
+  name: first_fetch
+  type: 0
+  required: false
+- name: feedExcludeOSIntel
+  display: Feed Exclude Open Source Intelligence
+  type: 8
+  defaultvalue: 'true'
+  additionalinfo: 'Whether to exclude Open Source Intelligence as part of the feed'
+  required: false
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed.
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  type: 15
+  defaultvalue: RED
+  required: false
+- additionalinfo: Supports CSV values.
+  display: Tags
+  name: feedTags
+  type: 0
+  required: false
+- name: feedExpirationPolicy
+  display: 'Feed Expiration Policy'
+  additionalinfo: Defines how expiration for an indicator created by the Feed will be managed
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  type: 17
+  required: false
+- name: feedExpirationInterval
+  display: 'Feed Expiration Interval'
+  additionalinfo: 'Defines the expiration date based on the number of days after an indicator is created / updated when the Feed Expiration Policy is set to `interval`'
+  type: 1
+  required: false
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  additionalinfo: 'How frequently the Feed should check Mandiant for new indicators'
+  type: 19
+  required: true
+- additionalinfo: Incremental feeds pull only new or modified indicators that have been sent from the integration. The determination if the indicator is new or modified happens on the 3rd-party vendor's side, so only indicators that are new or modified are sent to Cortex XSOAR. Therefore, all indicators coming from these feeds are labeled new or modified.
+  defaultvalue: 'true'
+  display: Incremental feed
+  hidden: true
+  name: feedIncremental
+  required: false
+  type: 8
+script:
+  commands:
+  - name: mandiant-get-indicators
+    arguments:
+    - description: The maximum number of indicators to fetch.
+      name: limit
```

Please add a `description` key to the `mandiant-get-indicators` command to fix the [pre-commit validation failure](https://github.com/demisto/content/actions/runs/10697530257/job/29654917907?pr=36047#step:8:875).

```suggestion
  - name: mandiant-get-indicators
    arguments:
    - description: The maximum number of indicators to fetch.
      name: limit
    description: Fetch indicators.
```

---

Pushed the change myself in https://github.com/demisto/content/pull/36047/commits/f3f3aba0911da1084edd1fe967857079d451cd6d.